packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte

```diff
@@ -265,64 +301,83 @@ function isContainerConnectionStatusInProgress(
-                    {#if container.lifecycleMethods.includes('stop')}
-                      <Tooltip tip="Stop" bottom>
-                        <button
-                          aria-label="Stop"
-                          class="{container.status !== 'started' ||
-                          isContainerConnectionStatusInProgress(provider, container)
-                            ? 'text-gray-700 cursor-not-allowed'
-                            : 'hover:text-gray-400'}"
-                          on:click="{() => stopContainerProvider(provider, container)}">
-                          <Fa class="mx-2.5 my-2" icon="{faStop}" />
-                        </button>
-                      </Tooltip>
-                    {/if}
-                    {#if container.lifecycleMethods.includes('delete')}
-                      <Tooltip tip="Delete" bottom>
-                        <button
-                          aria-label="Delete"
-                          class="{(container.status !== 'stopped' && container.status !== 'unknown') ||
-                          isContainerConnectionStatusInProgress(provider, container)
-                            ? 'text-gray-700 cursor-not-allowed'
-                            : 'hover:text-gray-400'}"
-                          on:click="{() => deleteContainerProvider(provider, container)}">
-                          <Fa class="mx-2.5 mr-5 my-2" icon="{faTrash}" />
-                        </button>
-                      </Tooltip>
-                    {/if}
+              {#if containerConnectionStatus.has(getContainerConnectionName(provider, container))}
+                {@const state = containerConnectionStatus.get(getContainerConnectionName(provider, container))}
+                {#if container.lifecycleMethods && container.lifecycleMethods.length > 0}
+                  <div class="mt-2 relative">
+                    <!-- TODO: see action available like machine infos -->
+                    <div class="flex bg-zinc-900 w-fit rounded-lg m-auto">
+                      {#if container.lifecycleMethods.includes('start')}
+                        <Tooltip tip="Start" bottom>
+                          <button
+                            aria-label="Start"
+                            class="ml-5 mr-2.5 my-2 {getStyleByState(state, 'start')}"
+                            on:click="{() => startContainerProvider(provider, container)}"
+                            disabled="{state.inProgress || state.status !== 'stopped'}">
+                            <LoadingIcon
+                              icon="{faPlay}"
+                              loadingWidthClass="w-6"
+                              loadingHeightClass="h-6"
+                              positionTopClass="top-1"
+                              positionLeftClass="left-[0.77rem]"
+                              loading="{state.inProgress && state.action === 'start'}" />
+                          </button>
+                        </Tooltip>
```

Comment:
(not a requested change, but I think it would help to have these lines as a component where you just provide the `state` field.